sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingStateStoreFormatCompatibilitySuite.scala

```diff
@@ -254,7 +253,7 @@ class StreamingStateStoreFormatCompatibilitySuite extends StreamTest {
   private def findStateSchemaException(exc: Throwable): Boolean = {
     exc match {
       case _: SparkUnsupportedOperationException => true
-      case _: InvalidUnsafeRowException => true
+      case _: SparkException => true
```

Comment:
Can we use StateStoreKeyRowFormatValidationFailure and StateStoreValueRowFormatValidationFailure here?

Reply:
Done